Improvement: using condition in when field (#686)
codemagic-ci-cd/codemagic-docs: content/yaml/yaml-conditional-run.md

> The current environment is accessible under the `env` variable.

Reviewer: ```suggestion
Information about the current environment is accessible under the `env` variable.
```

Reviewer: It would be nice to briefly mention the use case, e.g. so you can check branch information in the condition.

> For example, the build will continue if it was not triggered by a draft pull request update:

Reviewer: ```suggestion
For example, the build will continue if the triggering event was *not* a draft pull request update:
```

```yaml
workflows:
  build:
    name: Build on PR update
    triggering:
      events:
        - pull_request
    when:

>       condition: not event.pull_request.draft

Reviewer: should we still keep both the != 'master' and the "not a draft PR" example?

```

Webhook payload is accessible under the `event` variable. Note that `event` is not available if the build is started manually from the UI or by a schedule.

> Please check the structure of the webhook payloads that your git provider sends. **Webhooks** tab in application settings contains recently received webhooks.

Reviewer: Move this part to the top where you explain how or why use conditions.

> Please check the structure of the webhook payloads that your git provider sends. **Webhooks** tab in application settings contains recently received webhooks.

Reviewer: ```suggestion
You can check the structure of the webhook payloads that your git provider sends on the **Webhooks** tab in application settings.
```

> Note that in addition to `not` it is possible to use other logical operators in conditions, e.g. `and`, `==`.

Reviewer: Move this part right after the first sentence in this section and change the emphasis to: "You can use logical operators in conditions, e.g. not, and ...
